packages/form-plugin/src/directive.ts

```diff
@@ -27,6 +27,27 @@ export class FormDirective implements OnInit, OnDestroy {
         }
       });
 
+    // On first state change, sync form model, status and dirty with state
+    this._store
+      .select(state => getValue(state, `${this.path}`))
+      .pipe(takeUntil(this._destroy$), first())
```

Comment:
if we use first() here we don't need takeUntil since it will cleanup after the first item it gets. you could also use _store.selectOnce instead.

Reply:
I don't think fire on the first state change since it will always return a value immediately.

Reply:
You are right. In theory,  this._destroy$ may emit value before first value in store observable. 
It is safe to have a protection against possible value races.

Reply:
fair

Reply:
There is a problem. If we do it in sync manner, then UpdateFormValue will kill all the current state. 

I don't like how it works, but first we push state to form, then get a form value and push it back to state.

The reason is state model can be null or undefined, it can just have part of fields or it may has all the values there.